try:
    num = int("string")
except ValueError:
    raise ValueError("Oh! ValueError exception thrown")
